### PR TITLE
pr3429 - Allow for custom user modal sizes

### PR DESCRIPTION
**Pull request by RevanProdigalKnight**
Removed restriction to Bootstrap-native `modal-sm` and `modal-lg` size classes.

This change should not require any documentation changes

### CODE REVIEW

**Comment by chrisirhc** [comment]
@RevanProdigalKnight , your commit email is set to `@noreply.github.com`, did you intend that?

File: template/modal/window.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <div modal-render="{{$isRendered}}" tabindex="-1" role="dialog" class="modal fade" ng-class="{in: animate}" ng-style="{'z-index': 1050 + index*10, display: 'block'}" ng-click="close($event)">
-    <div class="modal-dialog" ng-class="{'modal-sm': size == 'sm', 'modal-lg': size == 'lg'}"><div class="modal-content" modal-transclude></div></div>
+    <div class="modal-dialog" ng-class="size ? 'modal-' + size"><div class="modal-content" modal-transclude></div></div>
```
Comment (chrisirhc) [inline template/modal/window.html]: Needs `: ''`.

**Comment by RevanProdigalKnight** [comment]
No, I did not intend that. I'm not entirely sure how to change that.

**Comment by chrisirhc** [comment]
See https://help.github.com/articles/setting-your-email-in-git/ .

File: template/modal/window.html (hunk 1 above)
Comment (RevanProdigalKnight) [inline template/modal/window.html]: Whoops, should be fixed now. You'd think after writing it right in the issue conversation I wouldn't make that sort of mistake...

**Comment by chrisirhc** [comment]
Thank you!

**Pull request closed without merging**